pydis_site/templates/events/pages/code-jams/the-code-style-guide.html

```diff
@@ -0,0 +1,257 @@
+{% extends "events/base_sidebar.html" %}
```

Comment:
Adding a comment on this line, but we should rename this file to be `code-style-guide.html` and drop the "the".